CHANGELOG.md

```diff
@@ -701,6 +703,17 @@ filename | sha256 hash
 
 [Documentation](https://docs.k8s.io) & [Examples](https://releases.k8s.io/release-1.6/examples)
 
+## Known Issues for v1.6.5
+
+* If you use the [GLBC Ingress Controller](https://github.com/kubernetes/ingress/tree/master/controllers/gce),
```

Comment:
I feel that "overwrite of manual edits to GCP Health Checks" captures the last point well enough. IMO